modules/issue_tracker/test/issue_tracker_test_plan.md

```diff
@@ -6,16 +6,20 @@ Issue Tracker Filter Form [Automation Testing]
 5. Test that the watching checkbox works correctly (issues that your userID is watching in issues_watching table)
 6. Check that links to issues in table are correct.
 7. Check that table sorts and displays additional pages correctly 
+8. Check that a user who does not have access to all centers can see all issues with a site set to NULL.
```

Comment:
this should say what permission it's referring to, since we have like 20 different access all site permissions..